Mount rootfs rw so we don't have to treat cases where users don't have a battery
for their odroid and their hwclock won't be accurate. A case like this would break boot.

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Florin Sarbu <florin@resin.io>

Index: odroid-xu3/autoboot.cmd
===================================================================
--- odroid-xu3.orig/autoboot.cmd
+++ odroid-xu3/autoboot.cmd
@@ -8,7 +8,7 @@ setenv fdt_high "0xffffffff"
 setenv macaddr "00:1e:06:61:7a:39"
 
 # --------------------------------
-setenv bootrootfs "console=tty1 console=ttySAC2,115200n8 root=/dev/mmcblk0p2 rootfstype=ext4 rootwait noinitrd"
+setenv bootrootfs "console=tty1 console=ttySAC2,115200n8 root=/dev/mmcblk0p2 rw rootfstype=ext4 rootwait noinitrd"
 
 # boot commands
 setenv bootcmd "fatload mmc 0:1 0x40008000 zImage; fatload mmc 0:1 0x44000000 exynos5422-odroidxu3.dtb; bootz 0x40008000 - 0x44000000"
